rest/api.go

```diff
@@ -520,6 +520,7 @@ func (h *handler) handleProfiling() error {
 			base.InfofCtx(h.ctx(), base.KeyAll, "... ending CPU profile")
 			pprof.StopCPUProfile()
 			h.server.CloseCpuPprofFile(h.ctx())
+			base.Audit(h.ctx(), base.AuditIDSyncGatewayProfiling, base.AuditFields{"profile_type": "cpu"})
```

Comment:
I think this case should have a file listed since this is writing a file. I thought maybe we should make file a mandatory but the `net/http/pprof` endpoints write to a response, so that doesn't make sense.

I do think there is something to consider the difference between cpu profile (stop) ("collect") vs the handler below:

```
base.Audit(h.ctx(), base.AuditIDSyncGatewayProfiling, base.AuditFields{"profile_type": "cpu (start)", "filename": params.File})
```

I thought about whether there should be an optional action that is "start" and "write", and how to reconcile those with the pprof handler that uses query parameters to take a profile over time.